getzim.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+if [ ! -e ".content" ]; then
+  echo "Downloading content list..."
+  curl -s http://wiki.kiwix.org/wiki/Content_in_all_languages > .content
```

Comment:
It did not work for me unless I enabled following redirect to HTTPS:
```suggestion
  curl -sL http://wiki.kiwix.org/wiki/Content_in_all_languages > .content
```

Reply:
Maybe just better use the https version https://wiki.kiwix.org/wiki/Content_in_all_languages